src/AddTodoForm.js

```diff
@@ -20,16 +21,14 @@ function AddTodoForm(props) {
 
     return (
         <form onSubmit={handleAddTodo}>
-            <label htmlFor="todoTitle" id="todoTitle">
-                Title
-            </label>
-            <input
-                type="text"
-                id="todoTitle"
-                name="title"
-                value ={todoTitle}
-                onChange = {handleTitleChange}
-            ></input>
+            <InputWithLabel
+            id = "id"
+            type = "text"
+            todoTitle = {todoTitle}
+            handleTitleChange={handleTitleChange}
+            children
```

Comment:
I think you misunderstood the instructions here. What you needed to do was add another prop called `label` and add the text 'Title` on it. After that, place it inside the label like this - `<label>{props.label}</label>`

Very crafty initial solution but we don't need to make it that complicated to get the title text to show up.